samples/cpp/benchmark_app/main.cpp

```diff
@@ -495,7 +495,7 @@ int main(int argc, char* argv[]) {
                     type_to_set = ov::element::u8;
                 }
 
-                auto& in = preproc.input(item.get_index());
+                auto& in = preproc.input(item.get_any_name());
```

Comment:
@fzhar `get_index()` returns index of port. It is incorrect usage, if we have other places where you used this method, we need to fix it.